platform/darwin/src/MGLStyleLayer.h

```diff
@@ -63,12 +63,12 @@ MGL_EXPORT
 @property (nonatomic, assign, getter=isVisible) BOOL visible;
 
 /**
- The maximum zoom level at which the layer gets parsed and appears.
+ The exclusive upper bound of the range of zoom levels within which a map would display the layer.
```

Comment:
I agree; the previous definition was easier to parse. I'm not convinced we actually need to make any documentation changes here. If anything we could simply remind the reader that this is a _floating point_ value, not an integer zoom level as they may be used to from raster-tile-based mapping libraries.